src/components/pages/collectRecordFormPages/BenthicPitForm/BenthicPitObservationTable.js

```diff
@@ -68,10 +68,15 @@ const BenthicPitObservationsTable = ({
   testId,
 }) => {
   const [observationsState, observationsDispatch] = observationsReducer
-  const [autoFocusAllowed, setAutoFocusAllowed] = useState(false)
 
   const { interval_start: intervalStart, interval_size: intervalSize } = formik.values
 
+  const handleAddObservation = () => {
```

Comment:
it makes sense in the observation tables. especially since deleteObservation is also there, would definitely rather those stay in the same section to be consistent